packages/react-art/src/ReactARTHostConfig.js

```diff
@@ -7,10 +7,18 @@
 
 import Transform from 'art/core/transform';
 import Mode from 'art/modes/current';
+import {enableNewReconciler} from 'shared/ReactFeatureFlags';
 import invariant from 'shared/invariant';
 
 import {TYPES, EVENT_TYPES, childrenAsString} from './ReactARTInternals';
 
+import {DefaultLanePriority as DefaultLanePriority_old} from 'react-reconciler/src/ReactFiberLane.old';
+import {DefaultLanePriority as DefaultLanePriority_new} from 'react-reconciler/src/ReactFiberLane.new';
+
+const DefaultLanePriority = enableNewReconciler
```

Comment:
To address @sebmarkbage's concern about leaking internals, can we move this to a reconciler export? Something like:

```js
import {DefaultEventPriority} from 'react-reconciler'
```

and 

```js
// ReactFiberReconciler.js
export const DefaultEventPriority =  enableNewReconciler ? DefaultLanePriority_new : DefaultLanePriority_old;
```

Then later we can change that export to be whatever we want, even if it requires us casting to `any` (though they're all just `number` anyway).

Reply:
So for example, a future PR would change ReactFiberReconciler to look like:

```js
export const DefaultEventPriority =  enableNewReconciler ? DefaultLane_new : DefaultLane_old;
```

without needing to change any of the renderers

Reply:
For default only?

Reply:
All of them

Reply:
Well all of the ones that are needed by the event system, I mean

Reply:
Can't do this after all, because it creates a circular import chain (Reconciler -> HostConfig -> Reconciler). There are probably smarter ways to break it off but for now I'm keeping the old structure (direct imports) but exporting some constants just for third-party renderers.